dashboard/config/locales/en.yml

```diff
@@ -428,6 +428,9 @@ en:
       legal_privacy: "Privacy Policy"
       legal_cookie_notice: "Cookie Notice"
       legal_tos: "Terms of Service"
+      videos: "Videos"
```

Comment:
The string in pegasus is here.
https://github.com/code-dot-org/code-dot-org/blob/c6759ecd99a16dc8c3b9626352bc118e9b985e7a/pegasus/cache/i18n/en-US.json#L1484